README.md

```diff
@@ -464,9 +466,7 @@ endif()
 
 ```cmake
 CPMAddPackage(
-  NAME lua
-  GIT_REPOSITORY https://github.com/lua/lua.git
-  VERSION 5.3.5
+  URI "gh:lua/lua@5.3.5"
```

Comment:
I think it would be a good idea to keep at least one example with the old API so users can easily understand how it works.

```suggestion
  NAME lua
  GIT_REPOSITORY https://github.com/lua/lua.git
  GIT_TAG v5.3.5
```